package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@r/flags",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Make sense. I'll merge with version set to 1.4.0 and then push a version-bump commit.